lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -16,6 +16,8 @@ setupRemoteFromTarget = (target, httpRequestType, form = null) ->
   options =
     httpRequestType: httpRequestType
     httpUrl: httpUrl
+    forceScrollReset: TurboGraft.getTGAttribute(target, 'force-scroll-reset')
+    forceHeadAssetInsertion: TurboGraft.getTGAttribute(target, 'force-head-asset-insertion')
```

Comment:
That's fair.

I might have been overdesigning the api, I just saw them as seperate things.

Maybe we can group them into a more semantic name as well?